src/jdk.compiler/share/classes/com/sun/tools/javac/comp/Flow.java

```diff
@@ -1023,8 +1026,15 @@ private List<ClassSymbol> baseClasses(TypeSymbol root) {
          * simplify the pattern. If that succeeds, the original found sub-set
          * of patterns is replaced with a new set of patterns of the form:
          * $record($prefix$, $resultOfReduction, $suffix$)
+         *
+         * useHashes: when true, patterns will be a subject to exact equivalence;
```

Comment:
- will be ~a~ subject
- two binding~s~ patterns

Reply:
Done, thanks!